frontend/src/components/ParticipantsList.js

```diff
@@ -44,118 +44,109 @@ const ParticipantsList = observer(() => {
 
   const rootStore = useContext(rootStoreContext)
   const participantsStore = rootStore.ParticipantStore
-  const [isLoading, setIsLoading] = useState(false)
-
-  // useEffect is a hook that gets called after every render/re-render.  Empty array second argument prevents it from running again.
-  useEffect(() => {
-    setIsLoading(true)
-    participantsStore.getParticipants()
-    participantsStore.getInsurers()
-    setIsLoading(false)
-  }, [participantsStore])
 
   const handleParticipant = (e, participant) => {
     participantsStore.setParticipant(participant)
+    participantsStore.setVisit(undefined)
   }
 
   return (
     <Fragment>
-      {isLoading ? (
-        <div>Loading ...</div>
-      ) : (
-        <div>
-          <Breadcrumbs separator="›" aria-label="breadcrumb">
-            <Link color="inherit" to="/">
-              Home
-            </Link>
-            <Typography color="textPrimary">Search Results</Typography>
-          </Breadcrumbs>
-          <Typography variant="h5" color="textPrimary">
-            Participants
-          </Typography>
-          <div className="participants">
-            <Table>
-              <TableHead>
-                <TableRow>
+      <div>
+        <Breadcrumbs separator="›" aria-label="breadcrumb">
+          <Link color="inherit" to="/">
+            Home
+          </Link>
+          <Typography color="textPrimary">Search Results</Typography>
+        </Breadcrumbs>
+        <Typography variant="h5" color="textPrimary">
+          Participants
+        </Typography>
+        <div className="participants">
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell>
+                  <Typography>PPID</Typography>
```

Comment:
This feels like it should be provided by the theme. Might be a good first issue to make

Reply:
Could be a good refactoring goal for next release.